testing/regress/ecl/sqnormds1_an.ecl

```diff
@@ -15,6 +15,7 @@
     limitations under the License.
 ############################################################################## */
 
+//nothor
 //nothorlcr
```

Comment:
Should this be deleted as no longer needed?

Reply:
I'm not sure.  What's target nothorlcr?

Reply:
It was used when we had two different thor implementations - one that used link counted rows (lcr) and one that didn't.
@AttilaVamos is nothorlcr ever changed anymore?

Reply:
@ghalliday no, I don't think.